public/app/features/search/reducers/manageDashboards.ts

```diff
@@ -46,13 +46,18 @@ const reducer = (state: ManageDashboardsState, action: SearchAction) => {
         }),
       };
     case MOVE_ITEMS: {
-      const { dashboards, folder } = action.payload;
-      const uids = dashboards.map((d: DashboardSectionItem) => d.uid);
+      const dashboards: DashboardSectionItem[] = action.payload.dashboards;
+      const folder: DashboardSection = action.payload.folder;
```

Comment:
You shouldn't need to type these if the action payload is correct type or?
```suggestion
      const dashboards = action.payload.dashboards;
      const folder = action.payload.folder;
```

Reply:
Probable relevant to the reply below, but there's no action creators here, just plain `dispatch` calls with `type` and `payload`, so no explicit typings on payload.